docs/source/installation.rst

```diff
@@ -17,6 +17,8 @@ There are several installation methods:
 
 Optionally, instructions are provided for :ref:`Docker users <docker_users>`.
 
+If you are a **Windows user**, please read `Windows Installation with WSL`_ first. 
```

Comment:
You'll have to update the internal reference here to point to the windows section; to do this you can follow the Docker example. You'll need to replace 
`Windows Installation with WSL`_ 
with 
:ref:`Windows Installation with WSL <windows_users>`
The "<windows_users>" part is the actual tag for where the link should go.
You can then add that tag above the windows section:
.. _windows_users:
(note the underscore _before_ windows, this is required by RST files)